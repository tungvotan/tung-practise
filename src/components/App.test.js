import React from 'react';
import configureStore from 'redux-mock-store';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import App from './App';

const setup = (setupProps = {}) => {
  const wrapper = shallow(<App/>);

  return {
    wrapper
  };
};

describe('App component', () => {
  test('renders without crashing', () => {
    const { wrapper } = setup();
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
