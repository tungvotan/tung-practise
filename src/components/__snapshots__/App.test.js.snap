// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders without crashing 1`] = `
<div>
  <Connect(AddTodo) />
  <Connect(TodoList) />
  <Footer />
</div>
`;
